src/main/java/it/eng/idsa/businesslogic/processor/receiver/ReceiverStaticResponseMessageProcessor.java

```diff
@@ -0,0 +1,65 @@
+package it.eng.idsa.businesslogic.processor.receiver;
+
+import java.io.File;
+import java.net.URI;
+import java.util.Base64;
+
+import javax.xml.datatype.XMLGregorianCalendar;
+import org.apache.camel.Exchange;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import de.fraunhofer.iais.eis.ArtifactResponseMessageBuilder;
+import de.fraunhofer.iais.eis.Message;
+import org.apache.camel.Processor;
+
+@Component
+public class ReceiverStaticResponseMessageProcessor implements Processor {
```

Comment:
Usually it is responsibility of dataApp to handle request, and read static file content.
Maybe we should move this logic to dataApp?

Reply:
Yes,we can move it to dataApp logic. Actually it isn't used in ECC, as it is commented out in the code can be removed. I used it to give a response with a pdf file in the payload